### ISSUE TITLE
Make moving evidence items between variants an option in the web interface

### ISSUE THREAD

**Issue by kkrysiak** [genome/civic-client#748]
Currently this requires an admin using the admin interface and is not tracked in the ways we are used to. 

It would be nice to be able to reassign an EID and not lose all of the associated content (comments, revision history, pending revisions, etc). Some of that may be maintained now but I don't think comments are.

**Comment by jmcmichael** [comment]
This should be straightforward to do on the client, but I'm not sure how much server-side work this would require. We should probably implement on the client as some kind of revision - @acoffman, would it be possible to include evidence re-assignment into the stream of revisions returned by the relevant endpoints?

**Comment by acoffman** [comment]
Yup! This is just one more field in the suggested change (in this case, variant_id). Creating a new variant for the purpose of moving an EID to it is slightly more complicated but totally doable if need be. Just reassigning to an existing Variant is trivial! I'll add it as a supported field on the backend for revisions.

**Comment by acoffman** [comment]
For a first pass we're going to support moving between already existing variants. Creating a new variant on the fly via an EID edit is likely going to be a v2 feature.
